graders/c/cgrader/cgrader.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, compiler="gcc"):
             self.data = json.load(file)
         self.compiler = compiler
 
-    def run_command(self, command, input=None, sandboxed=True, timeout=None, env=None):
+    def run_command(self, command, input=None, sandboxed=True, timeout=None, env=None):  # noqa: A002
```

Comment:
I decided to ignore these `input` warnings as it feels like a good name, and also something that a lot of downstream callers would use.

Reply:
And it matches the [`communicate()` parameter name](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate).